src/main/java/com/fcgl/madrid/points/service/UserTrophyService.java

```diff
@@ -1,22 +1,62 @@
 package com.fcgl.madrid.points.service;
 
 import com.fcgl.madrid.points.model.UserTrophy;
+import com.fcgl.madrid.points.model.response.GetUserTrophyResponse;
 import com.fcgl.madrid.points.repository.UserTrophyRepository;
+import com.fcgl.madrid.points.model.InternalStatus;
+import com.fcgl.madrid.points.model.StatusCode;
+import com.fcgl.madrid.points.model.ExceptionUtil;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.transaction.TransactionSystemException;
 
+import io.github.resilience4j.circuitbreaker.CallNotPermittedException;
+import io.github.resilience4j.circuitbreaker.annotation.CircuitBreaker;
+
+import javax.validation.ConstraintViolation;
+import javax.validation.ConstraintViolationException;
+
+import java.lang.Exception;
+import java.lang.Throwable;
+import java.lang.StringBuilder;
+
+import java.util.Set;
 import java.util.List;
+import java.util.ArrayList;
 
 @Service
 public class UserTrophyService {
-    private UserTrophyRepository userTrophyRepository;
+	private UserTrophyRepository userTrophyRepository;
+	private GetUserTrophyResponse getUserTrophyResponse;
+
+	@Autowired
+	public UserTrophyService(UserTrophyRepository userTrophyRepository) {
+		this.userTrophyRepository = userTrophyRepository;
+	}
+
+	public List<UserTrophy> findAll() {
+		return userTrophyRepository.findAll();
+	}
+
+	public GetUserTrophyResponse getUserTrophiesById(Long userId) {
+		getUserTrophyResponse.setUserTrophies(userTrophyRepository.getUserTrophiesById(userId));
+		return getUserTrophyResponse;
```

Comment:
As you currently have it `internalStatus` is set to null but this should not be the case.
This should be something along the lines of

```
List<UserTrophy> userTrophies = userTrophyRepository.getUserTrophiesById(userId);
return new GetUserTrophyResponse(InternalStatus.OK, userTrophies)
```